aeron-client/src/main/java/io/aeron/logbuffer/LogBufferDescriptor.java

```diff
@@ -217,7 +227,18 @@ public class LogBufferDescriptor
         offset += CACHE_LINE_LENGTH;
         LOG_DEFAULT_FRAME_HEADER_OFFSET = offset;
 
-        LOG_META_DATA_LENGTH = align(offset + LOG_DEFAULT_FRAME_HEADER_MAX_LENGTH, PAGE_MIN_SIZE);
+
+        offset += LOG_DEFAULT_FRAME_HEADER_MAX_LENGTH;
+
+        LOG_CHANNEL_URI_OFFSET = 1024;
+
+        LOG_META_DATA_LENGTH = align(LOG_CHANNEL_URI_OFFSET + SIZE_OF_INT + LOG_CHANNEL_URI_MAX_LENGTH, PAGE_MIN_SIZE);
+
+        System.out.println("offset:"+offset);
```

Comment:
Debug log.

Reply:
All the system.outs will removed in the final version of this PR.